### DISCUSSION TITLE
Stretching render to fit window size/fullscreen size

### DISCUSSION THREAD

**Discussion by SwagDeNoli** [parasyte/pixels#308] [Q&A] [answered]
How do I stretch the pixels pixel buffer to fit the screen (letter box/black bars on the screen) instead of having it be the default set size?

**Comment by parasyte** [comment, marked as answer]
There is an incomplete PR that shows how to do it with a custom renderer: #262 The comments also have a lot of information detailing why it hasn't been merged yet. The [README](https://github.com/parasyte/pixels/tree/fill-window/examples/fill-window#readme) included with the new example also contains some information about how it works and why you want to be careful about how you do non-integer scaling.
